### PR TITLE
[16.0][MIG] ddmrp_warning: Migration to 16.0

### PR DESCRIPTION
**Pull request by AlexPForgeFlow**
Migration to 16.0

[ForgeFlow](https://github.com/ForgeFlow)

### CODE REVIEW

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+odoo-addon-ddmrp@git+https://github.com/OCA/ddmrp.git@refs/pull/314/head#subdirectory=setup/ddmrp
```
Comment (LoisRForgeFlow) [inline test-requirements.txt]: Can you rebase and remove test requirements? This should be good to go after this.

**Review by LoisRForgeFlow — APPROVED** [review]
Code review LGTM

**Comment by BernatPForgeFlow** [comment]
LGTM! Code and functional review!

**Pull request merged**